src/components/Filters/types.ts

```diff
@@ -30,5 +30,11 @@ export interface Filter<V> {
   defaultValue: V | undefined;
 
   /** Renders the filter into either the page or the modal. */
-  render(value: V | undefined, setValue: (value: V | undefined) => void, tid: TestIds, inModal: boolean): JSX.Element;
+  render(
+    value: V | undefined,
+    setValue: (value: V | undefined) => void,
+    tid: TestIds,
+    inModal: boolean,
+    vertical: boolean,
```

Comment:
Wondering if we should maybe do a "placement"(?) property and combine these instead?

Reply:
Ah yeah, I like that...although it assumes the vertical place will _never_ put things in the modal...well, or actually that would be nice b/c then it would be inModal=true vertical=false...  So yeah, makes sense.

Want to do that in this PR or just as a sometime follow up?

Reply:
Since there is not rush for this to go in, I think I'd prefer to do it as part of this PR.

Reply:
Thought about it, and played around with alternative approaches, but I didn't think it was much cleaner than this. So will leave for now. Happy to revisit at another time.